Makefile

```diff
@@ -25,8 +18,5 @@ stop: ## Stop the server
 	docker-compose down
 
 test: ## Test the code
-	docker build -t service_php docker/php
-	$(MAKE) composer-install
-	docker run -it --rm -v "${PWD}/app:/app" service_php bin/phpunit
```

Comment:
It's easier to use your existing services. And here,  the *volume*  `/app:/app` seems wrong.